po/hi.po

```diff
@@ -0,0 +1,1546 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```

Comment:
Can you update the header with relevant information?

Reply:
Do I have to indicate my E-Mail in item #4?

Reply:
I mean the fourth line of the translation file.

Reply:
It's not mandatory, but it helps if another translator wants to get in touch in the future, I suppose.

Reply:
> It's not mandatory, but it helps if another translator wants to get in touch in the future, I suppose.

added E-mail.